External/AvalonTools/AvalonTools.h

```diff
@@ -16,43 +16,65 @@
 namespace AvalonTools {
 static int avalonSSSBits = 0x007FFF;
 static int avalonSimilarityBits = 0xF07FFF;
-RDKIT_AVALONLIB_EXPORT std::string getCanonSmiles(RDKit::ROMol &mol, int flags = -1);
-RDKIT_AVALONLIB_EXPORT void getAvalonCountFP(const RDKit::ROMol &mol,
-                      RDKit::SparseIntVect<boost::uint32_t> &res,
-                      unsigned int nBits = 512, bool isQuery = false,
-                      bool resetVect = true,
-                      unsigned int bitFlags = avalonSSSBits);
-RDKIT_AVALONLIB_EXPORT void getAvalonFP(const RDKit::ROMol &mol, ExplicitBitVect &res,
-                 unsigned int nBits = 512, bool isQuery = false,
-                 bool resetVect = true, unsigned int bitFlags = avalonSSSBits);
-RDKIT_AVALONLIB_EXPORT void getAvalonFP(const RDKit::ROMol &mol, std::vector<boost::uint32_t> &res,
-                 unsigned int nBits = 512, bool isQuery = false,
-                 bool resetVect = true, unsigned int bitFlags = avalonSSSBits);
-RDKIT_AVALONLIB_EXPORT unsigned int set2DCoords(RDKit::ROMol &mol, bool clearConfs = true);
-
-RDKIT_AVALONLIB_EXPORT std::string getCanonSmiles(const std::string &data, bool isSmiles,
-                           int flags = -1);
-RDKIT_AVALONLIB_EXPORT void getAvalonCountFP(const std::string &data, bool isSmiles,
-                      RDKit::SparseIntVect<boost::uint32_t> &res,
-                      unsigned int nBits = 512, bool isQuery = false,
-                      unsigned int bitFlags = avalonSSSBits);
-RDKIT_AVALONLIB_EXPORT void getAvalonFP(const std::string &data, bool isSmiles, ExplicitBitVect &res,
-                 unsigned int nBits = 512, bool isQuery = false,
-                 bool resetVect = true, unsigned int bitFlags = avalonSSSBits);
+
+namespace {
+void rdunused_var() { RDUNUSED_PARAM(avalonSimilarityBits); }
```

Comment:
There has to be a better way.  Perhaps 
```
const int
```

would work better than static here.

Reply:
I have the suspicion that it will still raise the warning in the implementation filese that include this header but don't use the variable.

Reply:
I think the issue is static here.  I also think this will end up in different translation units. Const int is fine in a header, I would be surprised if it is flagged as unused.

Reply:
How about we just have a declaration here and move the definition into a `.cpp` file?

Reply:
Moving the unnamed namespace that references avalonSimilarityBits  into .cpp should fix the issue as well.  

Basically the issue is that this is only used in the python wrapper so we could (a) move it there (b) reference avalonSimilarityBits  it in the avalantools lib or (c) (shudder) make it a define

Reply:
Using `const int` did the trick.